src/styles/blocks/card.scss

```
@@ -0,0 +1,106 @@
+
+.card {
+  width: 200px;
+  height: 408px;
+  background: $color-background;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__product {
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 0;
+  }
+
+  &__label {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+    width: 166px;
+    height: 36px;
+    margin-inline: 16px;
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    margin-top: 4px;
+    margin-left: 16px;
+  }
+
+  &__review {
+    margin-left: 17px;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__cost {
+    display: flex;
+    margin: 16px;
+    justify-content: space-between;
+  }
+
+  &__price {
+    width: 31px;
+    height: 18px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__money {
+    width: 50px;
+    height: 18px;
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &__button {
+    text-align: center;
+  }
+
+  &__buy {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    background-color: #00acdc;
+
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: $color-background;
+  }
+
+  &__buy:hover {
+    border: 1px solid #00acdc;
+    color: #00acdc;
+    background: $color-background;
+  }
```

```suggestion
  &__buy {
    width: 166px;
    height: 40px;
    border-radius: 5px;
    border: none;
    background-color: #00acdc;

    font-family: Roboto, sans-serif;
    font-size: 14px;
    font-weight: 700;
    line-height: 16px;
    text-align: center;
    color: $color-background;
    &__buy:hover {
    border: 1px solid #00acdc;
    color: #00acdc;
    background: $color-background;
    }
  }
```
make use of sass bem nesting